### PR TITLE
CORE-8016 - Refactoring RPC config keys and db table config (see corda-api pr 787)

### PR DESCRIPTION
**Pull request by pmck91**
Changes to runtime that reflect the refactoring in Corda-API: https://github.com/corda/corda-api/pull/787

### CODE REVIEW

**Comment by vkolomeyko** [comment]
@pmck91 - could you please refernce in the description `corda-api` PR which has changes necessary.

**Comment by pmck91** [comment]
> @pmck91 - could you please refernce in the description `corda-api` PR which has changes necessary.

Done

**Review by davidcurrie — DISMISSED** [review]
👍 for changes to `charts/corda`

File: gradle.properties [hunk 1]
```diff
@@ -40,7 +40,8 @@ bouncycastleVersion=1.72
 # Corda API libs revision (change in 4th digit indicates a breaking change)
 # Change to 5.0.0.xx-SNAPSHOT to pick up maven local published copy
 #cordaApiVersion=5.0.0.xxx-SNAPSHOT
-cordaApiVersion=5.0.0.613-beta+
+cordaApiVersion=5.0.0.614-alpha-1673969721173
```
Comment (vkolomeyko) [inline gradle.properties]: Humble reminder, this will need to be updated when Beta version of `corda-api` is produced.

**Review by vkolomeyko — DISMISSED** [review]
LGTM

@Tom-Fitzpatrick - FYI this might introduce conflicts to the entities mappings you were making in a separate PR.

**Review by conal-smith — DISMISSED** [review]
yeah lgtm

**Review by vkolomeyko — APPROVED** [review]

**Review by davidcurrie — APPROVED** [review]
👍

**Pull request merged**